### DISCUSSION TITLE
VcDrawings的drawings属性,能把"清除繪制結果"实例取消加載?

### DISCUSSION THREAD

**Discussion by stan88** [zouyaoji/vue-cesium#342] [Q&A]
您好,如題,
1.若要[清除繪制結果]這個實例不可見,能用drawings属性處理?
目前試是不行,還是有其他的方式可以達成?
2."清除繪制結果"实例或是编辑按钮的"移除所有節點"有事件觸發能抓取?
![2023-02-09_155140](https://user-images.githubusercontent.com/5283322/217750479-3dc5fe62-363d-409a-9448-8f743d8cb146.png)

**Comment by zouyaoji** [comment]
1、属性没法去掉删除按钮的。但可以通过slot插槽自定义按钮，然后去掉删除按钮。参考 https://zouyaoji.top/vue-cesium/#/zh-CN/component/analyses/vc-drawings例子中的  <template #body="drawingActionInstances">
2、已经提供了一个`editorEvt`事件，看看能用上吗？
https://github.com/zouyaoji/vue-cesium/blob/dev/packages/composables/use-drawing/use-drawing-point.ts#L445

**Reply by stan88** [reply]
感謝,editorEvt可用,
還有個疑問,為什麼 stopPropagation或是preventDefault無法停止刪除?
有什麼方法能按了"移除所有節點"後,可以取消?

**Reply by stan88** [reply]
您好,看了源碼
目前先以splice(index, 1)
可以達成目的,謝謝您
